Sources/VLCMovieViewController.m

```diff
@@ -1350,24 +1449,40 @@ - (void)toggleChapterAndTitleSelector
 - (void)toggleRepeatMode
 {
     [[VLCPlaybackController sharedInstance] toggleRepeatMode];
-    _multiSelectionView.repeatMode = [VLCPlaybackController sharedInstance].repeatMode;
+    #if !NEW_UI
+        _multiSelectionView.repeatMode = [VLCPlaybackController sharedInstance].repeatMode;
+    #else
+        _videoOptionsControlBar.repeatMode = [VLCPlaybackController sharedInstance].repeatMode;
+    #endif
 }
 
 - (void)toggleShuffleMode
 {
     _vpc.shuffleMode = !_vpc.isShuffleMode;
-    _multiSelectionView.shuffleMode = _vpc.isShuffleMode;
+    #if !NEW_UI
+        _multiSelectionView.shuffleMode = _vpc.isShuffleMode;
+    #endif
 }
 
 - (void)hideMenu
 {
-    [UIView animateWithDuration:.2
-                     animations:^{
-                         self->_multiSelectionView.hidden = YES;
-                     }
-                     completion:^(BOOL finished){
-                     }];
-    [self _resetIdleTimer];
+    #if !NEW_UI
+        [UIView animateWithDuration:.2
+                         animations:^{
+                             self->_multiSelectionView.hidden = YES;
+                         }
+                         completion:^(BOOL finished){
+                         }];
+        [self _resetIdleTimer];
+    #else
+        [UIView animateWithDuration:.2
+                         animations:^{
+                             self->_videoOptionsControlBar.hidden = YES;
```

Comment:
I believe we could avoid duplication here and make the check inside the animations block.